test/mockData/mockBooking.js

```diff
@@ -2,47 +2,47 @@ const accomodationBookings = [
   {
     accomodationId: '3dd3b34a-7554-425e-a688-36afda199619',
     roomId: '3dd3b34a-1254-495e-a688-36afda199619',
-    bookDate: '2019-09-15',
+    bookDate: '2020-09-15',
     tripId: 'ff10015c-1624-4490-9b1f-1a2cf0ee4493'
   },
   {
     accomodationId: '3dd3b34a-7554-425e-a688-36afda199619',
     roomId: '3dd3b34a-1254-495e-a688-36afda133619',
-    bookDate: '2019-09-15',
+    bookDate: '2020-09-15',
```

Comment:
This is a future bug. This test will fail on ```2020-09-15``` 😁

Reply:
Lol You're right!

Reply:
😆😆